src/status_im/contexts/wallet/add_account/create_account/key_pair_name/view.cljs

```diff
@@ -17,10 +17,23 @@
    :emoji        (i18n/label :t/key-name-error-emoji)
    :special-char (i18n/label :t/key-name-error-special-char)})
 
-(defn navigate-back
+(defn- navigate-back
   []
   (rf/dispatch [:navigate-back]))
 
+(defn- on-continue
```

Comment:
I think that naming functions by place when it is called isn't the best way, because names like `on-press`, `tap-handler` etc don't provide any meaningful information to the code reader and function better be called to reflect what is going inside of it. So I'd suggest renaming this to something like `select-workflow`, wdyt?

Reply:
Sure! I'll fix it!

Thanks for spotting it!